Web/CSS/CSS-Assignment-1/Css/Profile.css

```diff
@@ -0,0 +1,41 @@
+@CHARSET "ISO-8859-1";
+
+body {
+	background-color: #99CCFF;
+	margin:0px;
+}
+
+header {
+	border: solid 1px;
+	background-color: #6699FF;
+	text-align: center;
+}
+
+#maindiv div {
```

Comment:
create css with class names instead of using id. As id remains unique and can not be repeat to other component and by using class name we can reuse the css class for multiple components.